po/pt_BR.po

```diff
@@ -0,0 +1,1027 @@
+# Portuguese translations for devtoolbox package
+# Traduções em português brasileiro para o pacote devtoolbox.
+# Copyright (C) 2023 THE devtoolbox'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the devtoolbox package.
+# Rafael Fontenelle <rafaelff@gnome.org>, 2023.
```

Comment:
At the moment this file is not REUSE compliant. You have to add the SPDX-License line with any license to pass compliance:
```suggestion
# Rafael Fontenelle <rafaelff@gnome.org>, 2023.
# SPDX-License-Identifier: GPL-3.0-or-later
```

Reply:
Please consider this information as instruction for those willing to contribute with translation. Personally I'm not use to have this requirement.

Reply:
It's already written in the translation instructions but I'll make it more obvious.

Reply:
ah, totally missed it. Sorry.